crates/bevy_input/src/touch.rs

```diff
@@ -0,0 +1,49 @@
+use super::keyboard::ElementState;
+use crate::Input;
+use bevy_app::prelude::{EventReader, Events};
+use bevy_ecs::{Local, Res, ResMut};
+use bevy_math::Vec2;
+
+/// A finger on a touch screen device
+#[derive(Debug, Hash, PartialEq, Eq, Clone, Copy)]
+#[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
+pub struct Finger(pub u64);
```

Comment:
I also think I would prefer it if we didn't call this finger. I think TouchId is a more generic term